domain/provider.go

```diff
@@ -73,3 +96,26 @@ type ProviderType struct {
 	Name          string   `json:"name" yaml:"name"`
 	ResourceTypes []string `json:"resource_types" yaml:"resource_types"`
 }
+
+type AccessEntry struct {
+	Resource    string
```

Comment:
This one resource as in resource id or resource urn?

Reply:
this is resourceURN

Reply:
just realized that this field is not being used everywhere, removing it